csharp/samples/SampleEphReceiver/Program.cs

```diff
@@ -19,19 +19,12 @@ public class Program
 
         public static void Main(string[] args)
         {
```

Comment:
Can we leave this part in so that it is the same as the sender? Assuming we get the root key from the portal without an entity name specified?